src/libraries/System.Private.CoreLib/src/System/IO/BinaryReader.cs

```diff
@@ -453,6 +453,12 @@ public virtual byte[] ReadBytes(int count)
             return result;
         }
 
+        public virtual void ReadExactly(Span<byte> buffer)
```

Comment:
> The BinaryReader source files appear almost not have any tripple slash comments documenting the methods. I would of course be happy to add documentation, but I am unsure if the documentation of these methods is supposed to go somewhere else or if I am supposed to break with the existing "convention" of not documenting these methods.

Please add the tripple slash comments here. We have an internal porting process where we sporadically run a command line app that feeds the new comments to api docs. Sample outcome:  https://github.com/dotnet/dotnet-api-docs/pull/10210

You can use the wording from [Stream.ReadExactly](https://learn.microsoft.com/en-us/dotnet/api/system.io.stream.readexactly) as an inspiration.

Reply:
Thanks for the clarification, I have updated the method with a triple slash comment, based on the Stream.ReadExactly method as you suggested.